vizro-core/src/vizro/_vizro.py

```diff
@@ -47,6 +50,9 @@ def __init__(self, **kwargs):
             suppress_callback_exceptions=True,
             title="Vizro",
             use_pages=True,
+            # This is added here temporarily for testing. These stylesheets are required for certain dmc components.
+            # It may be better to include these stylesheets in the css folder.
+            external_stylesheets=dmc.styles.ALL,
         )
```

Comment:
We still need to let users define external_stylesheets like this: `Vizro(external_stylesheets=XXX).build(dashboard).run()`. So, it's better to append the stylesheet to the list instead of setting it here :) 

Let's just add `dmc.styles.DATES` to avoid loading unused stylesheets for now. If users need all `dmc` styles, they can still do it with `Vizro(external_stylesheets=dmc.styles.ALL).build(dashboard).run()`. This way, we only load what we need 👍 

Not sure if below will stay where it is, but for now, let's just add it there.

```suggestion
        )
        
        # Ensure external_stylesheets is a list and append the additional stylesheet
        external_stylesheets = self.dash.config.external_stylesheets
        self.dash.config.external_stylesheets = external_stylesheets if isinstance(external_stylesheets, list) else [
            external_stylesheets]
        self.dash.config.external_stylesheets.append(dmc.styles.DATES)
```